admin/controllers/service-manager.js

```diff
@@ -13,8 +13,11 @@ const checkWindowEditService = require('../services/check-window-edit.service')
 const sceService = require('../services/sce.service')
 const sceSchoolValidator = require('../lib/validator/sce-school-validator')
 const uploadedFileService = require('../services/uploaded-file.service')
+const serviceMessageProcessingService = require('../services/service-message-processing.service')
+const serviceMessageService = require('../services/service-message.service')
```

Comment:
A serviceMessageService?  Why not just call it a messageService?   I think the double `service` part is repetitive.

Reply:
updated with that.